src/test/java/seedu/address/testutil/TypicalAddressBook.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.testutil;
+
+import static seedu.address.testutil.TypicalPersons.getTypicalPersons;
+import static seedu.address.testutil.TypicalTasks.getTypicalTasks;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.person.Person;
+import seedu.address.model.task.Task;
+
+/**
+ * A utility class containing a list of {@code Person} and {@code Task} objects to be used in tests.
+ */
+public class TypicalAddressBook {
+
+    private TypicalAddressBook() {} // prevents instantiation
+
+    /**
+     * Returns an {@code AddressBook} with all the typical persons and tasks.
+     */
+    public static AddressBook getTypicalAddressBook() {
+        AddressBook ab = new AddressBook();
+        for (Person person : getTypicalPersons()) {
+            ab.addPerson(person);
+        }
+        for (Task task : getTypicalTasks()) {
+            ab.addTask(task);
+        }
+        return ab;
```

Comment:
```suggestion
        AddressBook ab = new AddressBook();
        ab.setPersons(getTypicalPersons());
        ab.setTasks(getTypicalTasks());
        return ab;
```

Since `getTypicalPersons()` returns a `List<Person>`, you can make use of predefined methods in `AddressBook` class to abstract away the logic here.